components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/mgt/services/ProcessManagementServiceSkeleton.java

```diff
@@ -918,4 +928,69 @@ private OMElement getProcessDefinition(ProcessConf pConf) throws ProcessManageme
         log.error(errMsg);
         throw new ProcessManagementException(errMsg);
     }
+
+    private static String getTryitURL(String serviceName, ConfigurationContext configurationContext) throws AxisFault {
+        AxisConfiguration axisConfig = configurationContext.getAxisConfiguration();
+        String wsdlURL = getWsdlInformation(serviceName, axisConfig)[0];
+        String tryitPrefix = wsdlURL.substring(0, wsdlURL.length() - serviceName.length() - 5);
+        if (!isServletTransport(axisConfig)) {
+            int tenantIndex = tryitPrefix.indexOf("/t/");
+            if (tenantIndex != -1) {
+                String tmpTryitPrefix = tryitPrefix.substring(tryitPrefix.substring(0, tryitPrefix.indexOf("/t/")).lastIndexOf("/"));
+                tryitPrefix = tryitPrefix.replaceFirst("//", "");
+                if (tryitPrefix.substring(0, tryitPrefix.indexOf("/services/")).lastIndexOf("/") > -1) {
+                    tryitPrefix = tryitPrefix.substring(tryitPrefix.substring(0, tryitPrefix.indexOf("/services/")).lastIndexOf("/"));
+                } else {
+                    tryitPrefix = tmpTryitPrefix;
+                }
+            } else {
+                tryitPrefix = configurationContext.getServiceContextPath() + "/";
+            }
+        }
+
```

Comment:
This code is copied from Java decompiled code. Please copy the logic from the original source code

Reply:
Fixed with https://github.com/wso2/carbon-business-process/pull/619/commits/f62ec675cec74fd2f337eb6165656e8b48547260